Turnstile-Ontology/99-Utils/CsvTemplates/Ingest-Threads.csv

```diff
@@ -0,0 +1 @@
+identifier_SoftwareDesign, identifier_SYSTEM_0, identifier_SYSTEM, 
```

Comment:
This is an example of a CSV file I'm not sure about exactly, but the trailing comma makes me wonder if something went wrong.

Reply:
Nothing wrong this matches the Spec.  The trailing comma was is just an artifact of when I generate the CSV template the script is not smart about only adding a comma when needed, it just for every property adds the "{Sparql_ID},".  I have the script to only include the comma between column headers.

Reply:
> * Need to change the comment for TOOL in AGENTS.sadl (we renamed the class).
> * Good to provide alias for DevelopmentPlan.sadl, CounterApplicationReviews.sadl, DevelopmentPlanData.sadl and maybe some other files.
> * Should we take this opportunity to align Turnstiles and Turnstile; note we have "Turnstiles is a SYSTEM, has identifier "Turnstile"."?

Updated TOOL notes to not be software specfic, added alias and harmonized them throughout the sadl files. updated Turnstile system definitions to be consistent in all sadl files for both planning and system design.